setup.cfg

```diff
@@ -20,7 +20,6 @@ disallow_incomplete_defs = True
 check_untyped_defs = True
 no_implicit_optional = True
 warn_redundant_casts = True
-warn_unused_ignores = True
```

Comment:
[Notes] `.github/workflows/checks-integration.yml` checks with this option (`warn_unused_ignores`) and unnecessary `# type: ignore` will be detected there.